pulumi/README.md

```diff
@@ -241,9 +241,17 @@ Errors like `SignatureDoesNotMatch: The request signature we calculated does not
 error: Running program '/Users/jafu6031/repos/folio/pulumi/folio' failed with an unhandled exception:
   Error: invocation of kubernetes:helm:template returned an error: failed to generate YAML for specified Helm chart: failed to pull chart: chart "bitnami/kafka" version "14.2.3" not found in https://charts.bitnami.com/bitnami repository
 ```
-
 This can mean that your local helm repo is out of data and doesn't have the latest charts. Try running `helm repo update`.
 
+## How to fully delete the database to start from scratch
+
+You can completely wipe out the db, by removing the deployment from the index.ts file, however the persistent volume will be stuck in the `Terminating` state. To get rid of this do:
+```
+kubectl get pv # To get names
+kubectl patch pv <pvname> -p '{"metadata":{"finalizers":null}}'
+```
+This is a well known issue without another solution. Reference: https://github.com/kubernetes/kubernetes/issues/69697
+
```

Comment:
I actually think that it would be better to drop the database and not try to remove the `pv`  or trying to delete the helm chart for the pg deployment. When trying to go back to a previous state (besides going nuclear with `pulumi destroy`) the above doesn't work well.